src/status_im/subs/wallet/wallet.cljs

```diff
@@ -464,22 +465,49 @@
  keep-operable-accounts)
 
 (rf/reg-sub
- :wallet/operable-accounts-without-watched-accounts
+ :wallet/operable-accounts
  :<- [:wallet/accounts-without-watched-accounts]
  keep-operable-accounts)
 
+(rf/reg-sub
+ :wallet/operable-addresses-tokens-with-positive-balance
+ :<- [:wallet/operable-accounts]
+ (fn [accounts]
+   (let [positive-balance-in-any-chain? (fn [{:keys [balances-per-chain]}]
+                                          (->> balances-per-chain
+                                               (map (comp :raw-balance val))
+                                               (some pos?)))]
+     (as-> accounts $
+       (group-by :address $)
+       (update-vals $ #(filter positive-balance-in-any-chain? (:tokens (first %))))))))
+
```

Comment:
Added this sub to return a map of `address` -> list of  its tokens with positive balance.

We need it to know if we should skip the from page or not